examples/source/user-consent/Advanced_User_Consent_Flow.html

```diff
@@ -85,24 +86,27 @@
 
   <!-- ## Basic usage -->
   <!--
-  The `amp-consent` component allows to specify the required user consent for this page. A CORS endpoints can be specified via the `checkConsentHref` attribute. This endpoint will be queried via a POST request to determine if the consent UI specified for it needs to be shown. The response should look like this:
+  The `amp-consent` component allows to specify the required user consent for this page using the required 'consentRequired' flag. A CORS endpoints can be specified via the `checkConsentHref` attribute along with `consentRequired` set to true. The `amp-consent` component then will check via a POST request if the consent UI needs to be shown. The response should look like this:
 
   ```json
   {
-    "promptIfUnknown": boolean (true/false)
+    "consentRequired": boolean (required),
```

Comment:
nit: remote.

Reply:
Not sure if we should add remote here, because we treat the `consentRequired` in the response as a boolean (!!response[`consentRequired`]). 

Also in the spec it says it should be a boolean https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/amp-consent.md#response